usr/share/rear/lib/nbkdc-functions.sh

```diff
@@ -0,0 +1,54 @@
+# locate NovaBACKUP DC client configuration, referenced by
+#
+# prep/NBKDC/default/40_prep_nbkdc.sh
+# skel/NBKDC/etc/scripts/system-setup.d/59-start-nbkdc-client.sh
+
+
+# Set the NovaBACKUP DC default directory
+NBKDC_DIR=/opt/NovaStor/DataCenter
```

Comment:
The actual backup agent can be installed anywhere (selectable during installation) and we want to locate it automatically. Just in case it cannot be located, I suppose a user can set it, for that the default.conf would make sense, yes.
I will add the line to the default.conf